model/clusters_mgmt/v1/cluster_resource.model

```diff
@@ -75,6 +75,17 @@ resource Cluster {
 		target MachinePools
 	}
 
+	// Initiates cluster hibernation; While hibernating a cluster will not consume any cloud provider infrastructure
+	// but will be counted for quota.
```

Comment:
This documentation isn't currently used for anything other than putting it in the OpenAPI documentation. But in the future I would like to use it to generate human readable documentation. In order to do that I would like to have in each piece of documentation a short sentence describing the thing. Then a dot, and then possible additional, longer, documentation. That dot will be used by the tool that generates the human readable documentation. So, can you reformat like this?

```
// Initiates cluster hibernation. While hibernating a cluster will not consume any cloud provider infrastructure
// but will be counted for quota.
```

Reply:
Thanks, done.